### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by ubitux**

### CODE REVIEW

File: libnopegl/src/backends/vk/glslang_utils.c [hunk 1]
```diff
@@ -118,7 +121,37 @@ int ngli_glslang_compile(int stage, const char *src, void **datap, size_t *sizep
         return NGL_ERROR_EXTERNAL;
     }
 
+/*
+ * We need this macro because until 11.11.0 the macro were completely broken
```
Comment (mbouron) [inline libnopegl/src/backends/vk/glslang_utils.c:125]: either `macros` or `was`

File: libnopegl/src/backends/vk/glslang_utils.c [hunk 2]
```diff
@@ -118,7 +121,37 @@ int ngli_glslang_compile(int stage, const char *src, void **datap, size_t *sizep
...
+ * See https://github.com/KhronosGroup/glslang/commit/6fdf03e4d1a6c2e9e0d8c69b122ff433cfe82225
+ */
+#define FIXED_GLSLANG_VERSION_GREATER_OR_EQUAL_TO(major, minor, patch) \
+    ((GLSLANG_VERSION_MAJOR) > (major) || ((major) == GLSLANG_VERSION_MAJOR && \
+    ((GLSLANG_VERSION_MINOR) > (minor) || ((minor) == GLSLANG_VERSION_MINOR && \
+     (GLSLANG_VERSION_PATCH >= (patch))))))
+
+#if FIXED_GLSLANG_VERSION_GREATER_OR_EQUAL_TO(11, 11, 0)
+    static const bool dbg = DEBUG_VK || DEBUG_GPU_CAPTURE;
```
Comment (mbouron) [inline libnopegl/src/backends/vk/glslang_utils.c:134]: Can we use bool now ?

**Review by mbouron — APPROVED** [review]
Other than my remarks, LGTM.

File: libnopegl/src/backends/vk/glslang_utils.c (hunk 2 above)
Comment (ubitux) [reply libnopegl/src/backends/vk/glslang_utils.c:134]: The fields are `bool` so… I can switch to `int` but this file unit has a dependency to these bool already so…

File: libnopegl/src/backends/vk/glslang_utils.c (hunk 1 above)
Comment (ubitux) [reply libnopegl/src/backends/vk/glslang_utils.c:125]: fixed

File: libnopegl/src/backends/vk/glslang_utils.c (hunk 2 above)
Comment (mbouron) [reply libnopegl/src/backends/vk/glslang_utils.c:134]: That's fine.

**Pull request merged**